app/controllers/collection_drafts_controller.rb

```diff
@@ -8,14 +8,14 @@ class CollectionDraftsController < BaseDraftsController
   layout 'collection_preview', only: [:show]
 
   def new
-    set_resource(CollectionDraft.new(user: current_user, provider_id: current_user.provider_id, draft: {}))
+    set_new_resource
```

Comment:
what resource is new? not descriptive enough

Reply:
well, that was a new name just for refactoring, we need to come up with a descriptive name.